specification/compute/resource-manager/Microsoft.Compute/stable/2022-07-02/DiskRP/diskRPCommon.json

```diff
@@ -449,6 +449,10 @@
     "SupportedCapabilities": {
       "type": "object",
       "properties": {
+        "diskControllerTypes": {
+          "type": "string",
```

Comment:
Is it not possible to define an enum type for it? Maybe check if PIR has defined it in their swagger.

Reply:
DiskControllerTypes is defined as an enum internally & in the shared client, (in all places except the External API Model).